src/components/CommentCard.jsx

```diff
@@ -1,10 +1,23 @@
 import "bootstrap/dist/css/bootstrap.min.css";
 import Card from "react-bootstrap/Card";
 import React from "react";
+import Button from "react-bootstrap/Button";
 import { CardHeader } from "react-bootstrap";
+import {deleteComment} from "../api"
 
-function CommentCard({comment}) {
-  // console.log(comment)
+function CommentCard({comment, setComments}) {
+  
+
+  function handleClick(event){
+    const commentIdToDelete = event.target.value
+    deleteComment(commentIdToDelete).then(()=>{
+      setComments((currComments)=>{
```

Comment:
You might want to consider error handling here - what happens if you're not able to delete a comment? 

At the moment the comment just stays, but you could display a message to the user to show there was a problem.